microproxy/utils.py

```diff
@@ -2,11 +2,18 @@
 from zmq.eventloop import zmqstream
 from zmq.log.handlers import PUBHandler
 from zmq.eventloop.ioloop import IOLoop
+from OpenSSL import SSL
 
 import logging
 import logging.config
 
 
+try:
```

Comment:
From mitmproxy, XD
We should run unittest depends on this flag that ignore alpn test if the environment not support alpn.